postman/schemas/index.yaml

```diff
@@ -17,11 +17,19 @@ paths:
       requestBody:
         required: true 
         content:
-          video/wav:
+          multipart/form-data:
             schema:
-              $ref: '#/components/schemas/AudioFile'
-          
-
+              type: object
+              properties:
+                AudioFile:
+                  type: object
+                  description: .wav file upload 
+                TextFile:
+                  type: object 
+                  description: .txt file upload
+                Recognizer:
```

Comment:
Could call this one 'language' instead because the recognizer is language-dependent.

Reply:
Although im almost certain it isn't one, I did avoid using 'language' in the code because I was concerned about reserved keywords and confusion between what 'language' we were actually talking about, but it would probably be fine to switch

Reply:
I can make that switch to language. I realize that these key values should match with the post requests Keys we are using in postman. So I will make that change. 

This also means that 'AudioFile' with be changed to 'audio' an TextFile with be changed to 'text'

Reply:
@edouarde1 @vnajack while we're at it, can we change characterSelect to avatar?

Reply:
hey sorry I did not this until now. Happy to do that change. Let's make sure that the change is reflected across the codebase. 

Would that be alright ? (e.g. in main at the moment the variable characterSelect should be changed to avatar)

Reply:
oh definitely! I really like the idea of using "avatar"!